src/connectors/captcha.ts

```diff
@@ -79,7 +112,24 @@ function success(data: string) {
     sentSuccess = true;
 }
 
-function info(message: string) {
-    parent.postMessage('info|' + message, parentUrl);
+function info(message: string | object) {
+    parent.postMessage('info|' + JSON.stringify(message), parentUrl);
+}
+
+async function watchHeight() {
+    const imagesDiv = document.body.lastChild as HTMLElement;
+    while (true) {
```

Comment:
Do we need to run this forever?

Reply:
we don't know when the captcha will be begun or solved. Are you wanting to stop after some number of seconds? Technically we could stop after a success, but I don't see a huge benefit to that since that _should_ mean the form is about to be submitted

Reply:
Gotcha, I didn't notice initially that we had to resize the iframe to fit the click.